dune/stuff/common/fixed_map.hh

```diff
@@ -76,22 +82,29 @@ public:
 private:
   friend class boost::iterator_core_access;
 
-  void increment() { index_++; }
+  void increment()
```

Comment:
I could not find an option to leave one-statement-blocks in one line. One could leave any blocks as one-liners if they already are, but that also includes blocks with an arbitrary amount of statements.